### PR TITLE
Adding files for DM1098EMB_R0 design.

### PR DESCRIPTION
**Pull request by Luxonis-David**
Added files for DM1098EMB design.

### CODE REVIEW

**Comment by Erol444** [comment]
@cafemoloko should we rename the product names in the readme file?

**Review by cafemoloko — APPROVED** [review]

**Comment by cafemoloko** [comment]
> @cafemoloko should we rename the product names in the readme file?

Sure

File: DM1098EMB_DepthAI_USB3/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+The reset button resets the Luxonis DepthAI SoM only, and the BOOT button forces USB boot for recovery mode in case DepthAI NOR flash boot header is corrupted during development.  
+
+The 5V LED indicates 5V power is present on the DM1098EMB. The PG LED indicates "power good" from the DepthAI SoM. The "RUN" LED indicates that the DepthAI SoM is not in reset.
+
+
+
+# Camera Usage
+
+This board is compatible with the [BG0249](https://github.com/luxonis/depthai-hardware/tree/master/BG0249_DepthAI_RGB_Camera), connected to the `RGB` FFC port and [BG0250TG](https://github.com/luxonis/depthai-hardware/tree/master/BG0250TG_DepthAI_Mono_Camera), connected to the `CAM_L` and/or `CAM_R` port.
```
Comment (cafemoloko) [inline DM1098EMB_DepthAI_USB3/README.md:75]:
```suggestion
This board is compatible with the [OAK-FFC-IMX378](https://github.com/luxonis/depthai-hardware/tree/master/BG0249_DepthAI_RGB_Camera), connected to the `RGB` FFC port and [OAK-FFC-OV9282](https://github.com/luxonis/depthai-hardware/tree/master/BG0250TG_DepthAI_Mono_Camera), connected to the `CAM_L` and/or `CAM_R` port.
```

**Review by cafemoloko — CHANGES_REQUESTED** [review]

**Comment by cafemoloko** [comment]
@Luxonis-David @Erol444 @GNejc - it would be best to add DM1098EMB to Hardware docs soon and keep the README brief in Github. I've started working on it ([PR here](https://github.com/luxonis/depthai-hardware/pull/101)), feel free to comment.

**Comment by Luxonis-David** [comment]
@cafemoloko / @Erol444 can we close this PR and merge to main?

**Review by Erol444 — APPROVED** [review]
LGTM, thanks David!

**Review by cafemoloko — APPROVED** [review]
Thanks, David!

**Pull request merged**